apps/review-host/update.sh

```
@@ -0,0 +1,62 @@
+# -m bounds the whole request: a service that accepts the connection and then
+# stops responding would otherwise hang here forever with no rollback.
+curl -sf -m 15 -H "Authorization: Bearer review-host-watchdog" \
+  http://127.0.0.1:48800/trpc/host.info >/dev/null || {
+    echo "[update] host-service did not answer after the restart"
+    roll_back
+  }
+
+# Answering on loopback is not the same as being reachable. The reviewer's phone
+# arrives over the relay, so keep the old build until the relay can see us.
+ORG="${REVIEW_ORG_ID:-9617bc8e-7f57-4af8-8b5e-586290ae536a}"
+for attempt in $(seq 1 12); do
```

Shellcheck (0.11.0)

[warning] attempt appears unused. Verify use (or export if used externally).

(SC2034)